common/changes/@rushstack/heft/user-danade-JestPlugin_2021-05-28-19-52.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/heft",
+      "comment": "Remove Jest plugin from Heft. To consume the Jest plugin, add @rushstack/heft-jest-plugin as a dependency and include it in heft.json",
```

Comment:
Please add migration instructions to [heft/UPGRADING.md](https://github.com/microsoft/rushstack/blob/master/apps/heft/UPGRADING.md).